src/store/Developers.js

```diff
@@ -0,0 +1,49 @@
+import API from "@/util/api";
+
+const state = {
+  developers: []
+};
+
+const mutations = {
+  SET_DEVELOPERS(state, developers) {
+    state.developers = developers;
+  }
+};
+
+const actions = {
+  async getDevelopers({ commit }) {
+    const developers = await API.getDevelopers().catch(e => {
+      console.error(e);
+
+      return [];
+    });
+
+    commit("SET_DEVELOPERS", developers);
+  },
+  getDeveloperByUsername(context, username) {
+    return state.developers.find(developer => developer.login === username);
+  },
+  async getRepositories({ commit }) {
```

Comment:
Any reason on why we have the action to load repositories here in the Developers store?

Reply:
Not sure if this is part of the PR changes though.

Reply:
Looks to be duplicated code

Reply:
Ops, it was left here by mistake. That was when I was splitting the developers and repositories into their stores.